Project/Assets/Prairie/Framework/Script/Twine/TwineNode.cs

```diff
@@ -31,24 +64,44 @@ public void OnGUI()
 		if (this.enabled && !this.isMinimized) {
 
 			float frameWidth = Screen.width / 3;
-			Rect frame = new Rect (10, 10, frameWidth, Screen.height);
+			Rect frame = new Rect (10, 10, frameWidth, Screen.height / 2);
 
 			GUI.BeginGroup (frame);
 			GUIStyle style = new GUIStyle (GUI.skin.box);
 			style.wordWrap = true;
 			style.fixedWidth = frameWidth;
 			GUILayout.Box (this.content, style);
+
+			if (isDecisionNode) {
+				GUIStyle decisionHintStyle = new GUIStyle (style);
+				decisionHintStyle.fontStyle = FontStyle.BoldAndItalic;
+
+				if (!isOptionsGuiOpen) {
+					GUILayout.Box ("Press Q to progress in the story...", decisionHintStyle);
+				} else {
+					GUILayout.Box ("Press Q to scroll, E to close, ENTER to choose", decisionHintStyle);
+				}
+			}
+
+			if (this.isOptionsGuiOpen) {
+				// Draw list of buttons for the possible children nodes to visit:
+				GUIStyle optionButtonStyle = new GUIStyle (GUI.skin.button);
+				optionButtonStyle.fontStyle = FontStyle.Italic;
+				optionButtonStyle.wordWrap = true;
+				selectedOptionIndex = GUILayout.SelectionGrid(selectedOptionIndex, childrenNames, 1, optionButtonStyle);
```

Comment:
Done. I grayed out the text for the inactive selection. Looks like this:

![](https://media.giphy.com/media/26gs7IcbWC7xF9ALC/giphy.gif)